README.md

```diff
@@ -20,7 +20,7 @@ sudo add-apt-repository ppa:mamarley/nvidia-dev
 
 The NVIDIA driver supports direct mode, meaning the HMD will not appear on your desktop, or if it does, the display will have to be turned off in xrandr before being able to use VR.
 
-**AMD graphics require a specific tree of the radv driver**. Use the **radv-wip-steamvr** branch of this repository: https://github.com/airlied/mesa. It doesn't support direct mode currently, so the HMD display will have to be positioned on your desktop in extended mode, and your system compositor disabled while using VR.
+**AMD graphics require a recent development version of the radv driver**. It must include the commit https://cgit.freedesktop.org/mesa/mesa/commit/?id=eaa56eab6da565c7bc84419c025c7c9bc90cf87a. For proper VK_KHR_external_semaphore_fd support Linux 4.13-rc1 or newer is required. It doesn't support direct mode currently, so the HMD display will have to be positioned on your desktop in extended mode, and your system compositor disabled while using VR.
```

Comment:
'Recent' is a relative term, it'll need to be changed again once the rest of the patches land in mesa.